dist/mapdc.js

```diff
@@ -34369,6 +34369,7 @@ function mapMixin(_chart, chartDivId, _mapboxgl) {
   // a way to re-enable it. Ideally, we'd just stop the event from firing the default actions, but
   // that doesn't seem to handle it. So here we are.
   function onMouseDownCheckForShiftToZoom(e) {
+    _map.boxZoom.disable();
```

Comment:
interacting with a lasso can re-enable the boxZoom filter, so we explicitly re-disable it when we click